packages/editor/src/store/selectors.js

```diff
@@ -297,10 +298,23 @@ export function getEditedPostAttribute( state, attributeName ) {
 			return getEditedPostContent( state );
 	}
 
+	// Fall back to saved post value if not edited.
 	if ( ! edits.hasOwnProperty( attributeName ) ) {
 		return getCurrentPostAttribute( state, attributeName );
 	}
 
+	// Merge properties are objects which contain only the patch edit in state,
+	// and thus must be merged with the current post attribute.
+	if ( EDIT_MERGE_PROPERTIES.has( attributeName ) ) {
+		// [TODO]: Since this will return a new reference on each invocation,
+		// consider caching in a way which would not impact non-merged property
+		// derivation. Alternatively, introduce a new selector for meta lookup.
+		return {
+			...edits[ attributeName ],
+			...getCurrentPostAttribute( state, attributeName ),
```

Comment:
Should we change the order here?

Reply:
I mean edits should override the current values?

Reply:
Yes 🤦‍♂️

Reply:
Also makes me wonder how we unset values, are we supposed to pass `null` as a value in these cases?

Reply:
> Should we change the order here?

Fixed in d711318.

> Also makes me wonder how we unset values, are we supposed to pass `null` as a value in these cases?

Unsetting would only occur when updating the post. It's worked this way since the beginning for any edit, so I'm not sure it's critical to change now.

I might imagine either assigning to `undefined` or assigning to the value in `currentPost` could unset the edit property.

Reply:
> I might imagine either assigning to `undefined` or assigning to the value in `currentPost` could unset the edit property.

It seems like a sensible, easy enhancement to consider. I'll make an issue for it.

Reply:
> > I might imagine either assigning to `undefined` or assigning to the value in `currentPost` could unset the edit property.
> 
> It seems like a sensible, easy enhancement to consider. I'll make an issue for it.

#12334